apps/dashboard/src/contract-ui/tabs/listings/components/list-form.tsx

```diff
@@ -213,14 +212,12 @@ export const CreateListingsForm: React.FC<CreateListingsFormProps> = ({
               new Date().getTime() +
                 Number.parseInt(formData.listingDurationInSeconds) * 1000,
             ),
-            minimumBidAmount: mulDecimalByQuantity(
-              formData.reservePricePerToken,
-              formData.quantity,
-            ),
-            buyoutBidAmount: mulDecimalByQuantity(
-              formData.buyoutPricePerToken,
-              formData.quantity,
-            ),
+            minimumBidAmount: (
+              BigInt(formData.reservePricePerToken) * BigInt(formData.quantity)
+            ).toString(),
+            buyoutBidAmount: (
+              BigInt(formData.buyoutPricePerToken) * BigInt(formData.quantity)
+            ).toString(),
```

Comment:
this won't work if values are decimals, probably need to parse the prices and the quantity can be simply bigint -> then multiply, boom

Reply:
yesser correct! I switched to the amount-in-wei. Did it on the plane, just got the chance to submit now